app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt

```diff
@@ -44,6 +44,21 @@ class NewCreatePinFragment :
         }
     }
 
+    private fun FragmentNewCreatePinBinding.initToolbar() = with(toolbar) {
```

Comment:
how bout' making it as an extension of Toolbar?

Reply:
Like this `private fun UiKitToolbar.initToolbar()`?

Reply:
yep

Reply:
Well, I don't mind